src/main/java/com/auth0/json/mgmt/client/Client.java

```diff
@@ -205,6 +205,14 @@ public Boolean isFirstParty() {
         return isFirstParty;
     }
 
+    /**
+     * Setter for whether this application is a first party client or not.
+     */
+    @JsonProperty("is_first_party")
+    public void setFirstParty(Boolean isFirstParty) {
```

Comment:
This should be really named setIsFirstParty or we're going to run into problems with code generation.

Reply:
@damieng I checked that and the rest of the methods in all the classes (even those other than Client) are not using "set IS x" but "set x".